### ISSUE TITLE
Websocket Connection fails if the host has an ipv6 address

### ISSUE THREAD

**Issue by EliasBinde** [Kounex/obs_blade#57]

**Comment by Kounex** [comment]
True, currently I'm expecting and working with IPv4 only. If I remember correctly, OBS WebSocket started out by only supporting IPv4 as well but is capable of doing IPv6 now. I will definitely take a look and see how to support it. Not that it's fundamentaly a difficult thing to do, but the fact that IPv6 addresses are so.. long would break several UI elements currently 🥲.

Added to the backlog for sure, thanks for raising this!
